easybuild/easyconfigs/s/SEED/SEED-2015.eb

```diff
@@ -0,0 +1,36 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   Copyright 2014-2017 adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# License::     Artistic-v2.0
+#
+# Notes::
+##
+
+easyblock = 'CmdCp'
+
+name = 'SEED'
+version = '2015'
+
+homepage = 'http://www.theseed.org/wiki/Main_Page'
+description = """ SEED is a software for clustering large sets of Next Generation Sequences (NGS) 
+with hundreds of millions of reads in a time and memory efficient manner. 
+Its algorithm joins highly similar sequences into clusters that can differ 
+by up to three mismatches and three overhanging residues. 
+"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

Comment:
@robqiao since the source is available, shouldn't we compile it ourselves with a proper toolchain?